### DISCUSSION TITLE
How to define custom shortcuts such as "aa" for "upright(bold(a))"?

### DISCUSSION THREAD

**Discussion by jademath** [typst/typst#365] [Q&A] [answered]
I'd like to define a shortcut so that "aa" expands to "upright(bold(a))" in math mode, just like the built-in shortcut "oo" for "infinity".

**Comment by laurmaedje** [comment, marked as answer]
```
#let aa = $upright(bold(a))$
```

**Reply by jademath** [reply]
I tried this already before. It only works if the  letter is at the end of the formula. Otherwise the formula will be corrupted.

**Reply by laurmaedje** [reply]
Can you show me what you tried?

**Reply by jademath** [reply]
It actually works now if I start from an empty document. I must have done something else previously.
